client/deposits/details/index.tsx

```diff
@@ -70,7 +70,7 @@ interface SummaryItemProps {
 	label: string;
 	value: string | JSX.Element;
 	valueClass?: string | false;
-	detail?: string;
+	detail?: string | JSX.Element;
```

Comment:
Does this align with the types for the component we are overriding (`SummaryNumber` from Woo core?

Reply:
The original SummaryItemNumber ( child within the summary list ) , does not have a detail prop.  It was introduced in the customized `SummaryListItem` for the Deposit Overview header.

Reply:
Unrelated to this PR, but I think we should avoid inheritance like this. We risk depending on implementation details, so our plugin is sensitive to changes in Woo core.